### DISCUSSION TITLE
Apexcharts react bar border-radius 4 angles?

### DISCUSSION THREAD

**Discussion by patchanop13** [apexcharts/apexcharts.js#2696] [Q&A]
I want border-radius 4 angles.My Apexcharts version is 3.28.3

![image](https://user-images.githubusercontent.com/56916752/136363962-643d1168-ffcc-4f76-815b-8c8660c9fee7.png)

**Comment by brianlagunas** [comment]
The border it not applied to the beginning of the series. Only the end.
